lib/fastlane/actions/mailgun.rb

```diff
@@ -26,13 +22,10 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
```

Comment:
This will invalidate all existing setups using this integration, right? Is there a reason you renamed `mailgun_sandbox_postmaster ` to `postmaster`?